src/components/Bar.jsx

```diff
@@ -69,22 +71,23 @@ class Bar extends Component {
           <sup class='coz-bar-hide-sm coz-bar-beta-status'>{t('beta')}</sup>
         </h1>
         <hr class='coz-sep-flex' />
+        {__TARGET__ !== 'mobile' && !isPublic &&
+          <div>
+            <button class='coz-bar-burger' onClick={this.toggleDrawer} data-icon='icon-hamburger'>
+              <span class='coz-bar-hidden'>{t('menu')}</span>
+            </button>
+            {claudyConfig &&
+              <Claudy config={claudyConfig} usageTracker={usageTracker} onToggle={this.toggleClaudy} opened={claudyOpened} />
+            }
+          </div>
+        }
       </div>
     )
```

Comment:
I see these variables are used in further commits, so I review a out-of-date commit. Hard to review with chopped commits.